packages/tasit-account/.babelrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
For the record, I usually use .js config files since I think most projects (like eslint as well) are pushing people in that direction these days. But that's not worth changing until we happen to be adding a new config for something else.

Reply:
noted!